integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -493,6 +493,23 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
           showHistory.isDefined)
     }
 
+
+  protected lazy val cli: Parser[LogicalPlan] =
+    (SHOW ~> SUMMARY ~> FOR ~> TABLE) ~ (ident <~ ".").? ~ ident ~
```

Comment:
Change `(SHOW ~> SUMMARY ~> FOR ~> TABLE) ~ (ident <~ ".").?` to `(SHOW ~> SUMMARY ~> FOR ~> TABLE) ~> (ident <~ ".").?` as we dont want the output of show summary for table

Reply:
changed